modules/ding_mobilesearch/ding_mobilesearch.module

```diff
@@ -143,12 +145,12 @@ function ding_mobilesearch_ding_mobilesearch_node_export_mapping($node) {
       ),
       'created' => array(
         'name' => t('Created'),
-        'value' => $node->created,
```

Comment:
This change is added multiple places. Date/timezone handling is generally complex. How about moving the handling to a central function à la `ding_mobilesearch_format_date($timestamp, $timezone)`?